### DISCUSSION TITLE
Welcome to lear Discussions!

### DISCUSSION THREAD

**Discussion by thorwolpert** [bcgov/lear#1373] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by severinbeauvais** [comment]
Hey there.
Let's see what adding a comment does...
🐯

**Reply by severinbeauvais** [reply]
So I've been auto-added as a participant. Neat.

This looks like a thread under the main discussion, but I wonder if new discussions should be created for new topics?
